MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,311 @@
+// swiftlint:disable all
+
 import UIKit
 
-final class MovieQuizViewController: UIViewController {
-    // MARK: - Lifecycle
+final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate {
+    
+    @IBOutlet var activityIndicator: UIActivityIndicatorView!
```

Comment:
Не забывайте модификаторы
```suggestion
    @IBOutlet private var activityIndicator: UIActivityIndicatorView!
```